src/docs/howto-payments/enable-payment-intents-in-ftw/index.md

```diff
@@ -5,8 +5,8 @@ updated: 2020-08-11
 category: how-to-payments
```

Comment:
This may well be out of the scope of this project, but I suggest renaming (and reshaping) this article into something along the lines of "how does the checkout page work". AFAIK originally this guide was here to help users using an older version of FTW after Stripe introduced payment intents. We have that info in the legacy docs if that is relevant for anyone anymore

Reply:
That's a good idea, I've now added a Whimsical card about it. Agree that it's outside the scope of this project.